### PR TITLE
changed styling of uploads page

### LINKED ISSUE #3660: disordered uploads page in mobile phones
WhatsApp Image 2020-01-13 at 11 24 42 PM

### PR DESCRIPTION
**Pull request by Manasa2850**
## What this PR does 
< Fixes #3660 >

## Screenshots
Same as the ones in https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/3661

### CODE REVIEW

**Comment by Manasa2850** [comment]
@ragesoss this is fine right?

**Comment by ragesoss** [comment]
@Manasa2850 yes, thanks! I'll probably be able to test it out and either merge or give feedback next week.

**Comment by Manasa2850** [comment]
@ragesoss sure! 
I haven't made any new changes. It's just the ones I had done previously.

**Comment by Manasa2850** [comment]
@ragesoss I'm currently resolving pr 3649. Let me know if I need to make any changes to this as well.

File: app/assets/stylesheets/modules/_selects.styl [hunk 1]
```diff
@@ -40,6 +40,7 @@
     background url('../images/sort.svg') no-repeat 48% white
     -webkit-appearance none
     text-indent -200px
+    margin-top -10px
```
Comment (ragesoss) [inline app/assets/stylesheets/modules/_selects.styl]: This is going to affect every sort-select, not just the one in the screenshot.

I think this adjustment either needs to be done in a way that just affects that uploads page, or in a way that address the same issue for every view that includes a sort-select without any code specific to uploads.

Comment (Manasa2850) [reply app/assets/stylesheets/modules/_selects.styl]: @ragesoss can I create a new className element for just the uploads page and make styling changes in that?

Comment (ragesoss) [reply app/assets/stylesheets/modules/_selects.styl]: No, that will lead to more styling trouble later one, if/when we try to adjust the styles of those selects and it leaves just one of them unchanged. Doing it in a way that would be beneficial for all the sort-selects is much preferable to a fix that just deals with this one.

Comment (Manasa2850) [reply app/assets/stylesheets/modules/_selects.styl]: I had a look at most of the sort-select buttons and it's only in the mobile view that there are layout issues. So I'll add **margin-top -10px** just to the mobile version.
Is that fine @ragesoss ?

Comment (Manasa2850) [reply app/assets/stylesheets/modules/_selects.styl]: Applying that code will not create any disorientation in other pages that have sort-select

Comment (ragesoss) [reply app/assets/stylesheets/modules/_selects.styl]: Okay. Can you include screenshots of before / after for both the uploads page and one of the other pages with a sort-select?

**Comment by Manasa2850** [comment]
Desktop (before)
desktop(before)

Desktop (after)
desktop(after)

Mobile (before)
mobile(before)

Mobile (after)
mobile(after)

Other page with a sort-select (before)
otherpage(before)

Other page with a sort-select (after)
other page(after)

**Comment by Manasa2850** [comment]
@ragesoss do I need to make any changes to this? If not, I'll start working on pr #3375

**Comment by ragesoss** [comment]
This looks good.

**Pull request merged**